pkg/messaging/direct_messaging_test.go

```diff
@@ -219,9 +227,11 @@ func TestInvokeRemote(t *testing.T) {
 		require.NoError(t, err)
 
 		assert.Equal(t, "🐱", string(pd.Message.Data.Value))
+		assert.Equal(t, routines, runtime.NumGoroutine())
```

Comment:
I wonder how reliable this test will be, since there could perhaps be other background goroutines started at random?

Reply:
Indeed, let's remove them.